packages/router-generator/src/generator.ts

```diff
@@ -462,6 +462,7 @@ export async function generator(config: Config) {
     '// @ts-nocheck',
     '// noinspection JSUnusedGlobalSymbols',
     '// This file is auto-generated by TanStack Router',
+    ...config.prependToRouteTreeFile,
```

Comment:
Do we want this to actually be prepended to the very top? or should it come after the stuff we are shipping in the file?